wormhole-connect/src/config/mainnet.ts

```diff
@@ -92,6 +92,17 @@ export const MAINNET_NETWORKS: NetworksConfig = {
     automaticRelayer: false,
     maxBlockSearch: 2000,
   },
+  sui: {
+    ...chains.sui!,
+    displayName: 'Sui',
+    explorerUrl: 'https://explorer.sui.io/',
+    explorerName: 'Sui Explorer',
+    gasToken: 'SUI',
+    chainId: 0,
+    icon: Icon.SUI,
+    automaticRelayer: false, // TODO: true
+    maxBlockSearch: 0, // TODO: what is this?
```

Comment:
It's for searching for the Redeem event for automatic transfers.  RPCs usually have a limit of how far back you can search